examples/demo.rs

```diff
@@ -958,11 +958,10 @@ fn draw_button<T: Renderer>(
     paint.set_text_align(Align::Left);
     paint.set_text_baseline(Baseline::Middle);
 
-    let tw = if let Ok(layout) = canvas.measure_text(0.0, 0.0, text, &paint) {
-        layout.width()
-    } else {
-        0.0
-    };
+    let tw = canvas
+        .measure_text(0.0, 0.0, text, &paint)
+        .map(|layout| layout.width())
+        .unwrap_or(0.0);
```

Comment:
I think this can be simplified:

```suggestion
        .map_or(0.0, |layout| layout.width())
```

Reply:
Thanks, I didn't know about that yet. Clippy already told me about such simplifications multiple times.